CHANGELOG.md

```diff
@@ -4,13 +4,28 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-
-## [Unreleased](https://github.com/ACWI-SOGW/well_registry_management/compare/wellregistry-0.1.0...master)
-### Fixed
--   Fixed the Delete button styling so button is full height.
-
-## [0.1.0](https://github.com/ACWI-SOGW/well_registry_management/tree/wellregistry-0.1.0) - 2020-11-05
-
+##[Unreleased](https://github.com/ACWI-SOGW/well_registry_management/tree/master)
```

Comment:
You are still overwriting the changes that I put in when I did a release. Please just add your change underneath the 0.1.0 tag

Reply:
fixed